<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameProjectWindow</class>
 <widget class="QWidget" name="RenameProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename project</string>
  </property>
  <widget class="QLabel" name="newNameLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newNameLineEdit">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>25</y>
     <width>235</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
